interfaces/Portalicious/src/app/components/page-layout/components/project-menu/project-menu.component.html

```diff
@@ -0,0 +1,6 @@
+<div
+  class="w-full bg-white px-6 pt-2 lg:px-24 [&_.p-menuitem-link]:border-transparent [&_.p-tabmenu-nav]:border-transparent [&_.p-tabmenuitem]:border-transparent"
```

Comment:
Not happy about this notation, I feel this is not the way to go but can't figure out the right way to do it.